components/blitz/resources/omero/api/IRenderingSettings.ice

```diff
@@ -20,30 +20,341 @@ module omero {
     module api {
 
         /**
-         * See <a href="http://downloads.openmicroscopy.org/latest/omero5.2/api/ome/api/IRenderingSettings.html">IRenderingSettings.html</a>
+         * Provides method to apply rendering settings to a collection of
+         * images.
+         * All methods will receive the id of the pixels set to copy the
+         * rendering settings from.
          **/
         ["ami", "amd"] interface IRenderingSettings extends ServiceInterface
             {
+                /**
+                 * Checks if the specified sets of pixels are compatible.
+                 * Returns <code>true</code> if the pixels set is valid,
+                 * <code>false</code> otherwise.
+                 *
+                 * @param pFrom The pixels set to copy the settings from.
+                 * @param pTo The pixels set to copy the settings to.
+                 * @return See above.
+                 **/
                 idempotent bool sanityCheckPixels(omero::model::Pixels pFrom, omero::model::Pixels pTo) throws ServerError;
+
+                /**
+                 * Returns the default rendering settings for a given pixels
+                 * for the current user.
+                 *
+                 * @param pixelsId The Id of the <code>Pixels</code>
```

Comment:
This file makes fairly liberal use of code blocks to highlight class names, etc.